doc/articles/migrating-from-previous-releases.md

```diff
@@ -6,6 +6,10 @@ uid: Uno.Development.MigratingFromPreviousReleases
 
 This article details the migration steps required to migrate from one version to the next when breaking changes are being introduced.
 
+## Uno Platform 5.4
+
+Uno Platform 5.4 does not contain breaking changes that require attention when upgrading from previous versions.
```

Comment:
@jeromelaban I imagine this **Upgrading from previous releases** section should include details from other related packages as well for the upgrade to Uno.Sdk 5.4, like uno.extensions for example.
cf. my comment here: https://github.com/unoplatform/uno.extensions/pull/2557#discussion_r1773801076

Reply:
Sure, let's add content there if relevant.

Reply:
@ajpinedam I will let you help to adjust the migrating section here to add a link to extensions migrating details based on your current in-progress PR https://github.com/unoplatform/uno.extensions/pull/2557

Reply:
The xref is now set, it will auto update when the hash from extensions is updated.